docs/answerfile.md

```diff
@@ -237,12 +237,18 @@ If `protov6` is static then the following elements must be present:
 
 #### Root password
 
-Specifies the root password. The value `!!` and a type of "hash" defers setting a password until first boot. Default: type="hash", `!!`.
+Specifies the root password. The value `!!` and a SHA-512 "hash" defers setting a password until first boot. Default: type="plaintext", `!!`.
```

Comment:
The change is not accurate to me. The default when no `root-password` entry is specified at all *is* type="hash" + value "!!", which means "ask at first boot", as the original sentence said.

I think you mean that when type= is not specified, then it's plaintext. I'm not in favor of adding this as this encourages not defining type when what the doc currently says is it should be defined, either with value "plaintext" or with value "hash".

Reply:
You are correct. When type is not specified default is plaintext.

Reply:
I will revert it to what it was but leave the SHA-512 hash decscription.